annotationsx/settings/aws.py

```diff
@@ -168,6 +173,10 @@
 
 # should be changed depending on platform roles, these are set up for edX
 ADMIN_ROLES = ['urn:lti:instrole:ims/lis/Administrator', 'urn:lti:role:ims/lis/Instructor', 'urn:lti:role:ims/lis/TeachingAssistant']
+
+#TODO: check if this is what we want
+ADMIN_ROLES += ['Administrator', 'Instructor', 'TeachingAssistant']
+
```

Comment:
Note: the ADMIN_ROLES defined by the LTI specification can be found here: 

http://www.imsglobal.org/lti/blti/bltiv1p0/ltiBLTIimgv1p0.html#_Toc261271984

The "Administrator" role is sent over as **urn:lti:instrole:ims/lis/Administrator** I believe, so you might want to check that.